src/Modal/README.md

```diff
@@ -38,7 +38,30 @@ Table below contains all types of the props available in the Modal component.
 ### Functional specs
 * Also, if you want to select the Close Button element for testing purposes, use [data-test="ModalCloseButton"] selector.
 
-
+* If you need to set scrollTop position of the Modal component, you can use instance of the component and method `setScrollPoint` like this:
```

Comment:
... you can use instance of the component and method `setScrollPosition` please 😉